src/test/scala/com/cognite/sdk/scala/v1/StringDataPointsTest.scala

```diff
@@ -4,17 +4,28 @@ import java.util.UUID
 
 import com.cognite.sdk.scala.common.{SdkTest, StringDataPointsResourceBehaviors}
 
+import scala.util.Random
+
 class StringDataPointsTest extends SdkTest with StringDataPointsResourceBehaviors[Long] {
-  override def withStringTimeSeriesId(testCode: Long => Any): Unit = {
-    val timeSeriesId = client.timeSeries.createFromRead(
-      Seq(TimeSeries(name = s"string-data-points-test-${UUID.randomUUID().toString}", isString = true))
-    ).head.id
+  override def withStringTimeSeriesId(testCode: (Long, Long) => Any): Unit = {
+    val timeSeriesId = client.timeSeries
+      .createFromRead(
+        Seq(
+          TimeSeries(
+            name = s"string-data-points-test-${UUID.randomUUID().toString}",
+            isString = true
+          )
+        )
+      )
+      .head
+      .id
+    val missingTimeSeriesId = new Random().nextLong() * -1
     try {
-      val _ = testCode(timeSeriesId)
+      val _ = testCode(timeSeriesId, missingTimeSeriesId)
     } finally {
       client.timeSeries.deleteByIds(Seq(timeSeriesId))
     }
   }
 
-  it should behave like stringDataPointsResource(client.dataPoints)
+  (it should behave).like(stringDataPointsResource(client.dataPoints))
```

Comment:
We don't format ScalaTest code like this. I wonder if you can configure IntelliJ to not format on save in the test directory?